src/components/HelperHelp.js

```diff
@@ -19,10 +20,13 @@ import {
 } from '../constants.js'
 
 export const HelperHelp = connect(({ settings, showQueue }) => ({
-  queue: localStorage.queue,
   settings,
   showQueue
-}))(({ queue, settings, showQueue, dispatch }) =>
+}),
+(dispatch) => ({
+  queue: getQueue(dispatch)
```

Comment:
If I am following you correctly here, you are just trying to get the current value of the queue to pass to the component? If that's the case, that is what the `connect` function is for. Simply add `queue` to the deconstructed argument (along with `settings` and `showQueue`) and pass it along to the component.